tests/cnf/core/network/internal/netconfig/config.go

```diff
@@ -158,6 +159,19 @@ func (netConfig *NetworkConfig) GetSwitchInterfaces() ([]string, error) {
 	return envValue, nil
 }
 
+// GetPrimarySwitchInterfaces checks the environmental variable ECO_CNF_CORE_NET_PRIMARY_SWITCH_INTERFACES
```

Comment:
Do we need to think about clusters with 3 workers? So 6 ports?

Reply:
nope, we gonna test only 2 of them.  We already have  the similar variable  ECO_CNF_CORE_NET_SWITCH_INTERFACES